app/services/reports/facility_progress_service.rb

```diff
@@ -29,6 +30,7 @@ def initialize(facility, period)
           .for_region(region)
           .where("month_date > ?", 6.months.ago.beginning_of_month.to_date)
           .load
+      @yearly_facility_data = FacilityYearlyFollowUpsAndRegistrationsQuery.new(facility, @current_user).call.values
```

Comment:
Looks like we're jumping through some extra hoops with `.values` here to make the data structure consistent with monthly and daily data. Perhaps we can update the query class to return an array/collection directly instead.